src/__test__/server.test.ts

```diff
@@ -12,7 +12,7 @@ describe("GET /api/health", () => {
         await fastify.close();
     });
 
-    it("should return status OK", async () => {
+    it("should return status 200 and token", async () => {
```

Comment:
and token? We dont send token here? Dont think we should anyways.

```suggestion
    it("should return status 200", async () => {
```

Reply:
Brainfart